doc/page-features.h

```diff
@@ -2,6 +2,52 @@
 
 \page Features
 
+
+\section async Asynchronous methods
+
+In addition to standard methods, which block the caller until the method execution is finished and provide an immediate output value, Facelift
+supports asynchronous methods. Asynchronous methods have been introduced to support operations which potentially last a long time, or which are
+not able to produce an output value immediately.
+
+\subsection async-sub1 Declaration
+
+Methods can be marked as asynchronous using the "async" decorator, as in the following QFace definition:
+
+\code
+module facelift.example 1.0;
+
+interface MyInterface {
+
+    @async : true
+    int myAsyncMethod(int inputArgument);
+
+}
+\endcode
+
+\subsection async-sub2 Usage from QML UI code
+In order for the method's return value to be provided asynchronously, the method needs to be called with a javascript function object as argument, in addition to the
+"normal" argument. The javascript function should accept an argument, unless the method is defined as returning "void".
+\code
+    myInterface.myAsyncMethod(integerInputArgument, function(integerResult) {
+        print("myAsyncMethod finished execution with result " + integerResult);
+    });
+\endcode
+
+\subsection async-sub3 C++ method implementation
+In order for a C++ implementation to provide its output value asynchronously, an asynchronous method takes an additional argument. That argument defines
+a "operator(...)" method, which means it can be called as if it was a std::function. Although it can be called synchronously, that object would typically
```

Comment:
ellipses are confusing - it is operator()
s/defines a "operator(...)" method/is a functor (it defines operator()) ?
s/as if it was a std::function/like a function ?